book.rb

```diff
@@ -0,0 +1,16 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals.push(rental)
+    rental.book = self
```

Comment:
- [ ] with the previous comment I meant that the rental class should be initialized similarly to below:
```
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```